rush.json

```diff
@@ -356,6 +356,13 @@
       "shouldPublish": true,
       "versionPolicyName": "prerelease-monorepo-lockStep"
     },
+    {
+      "packageName": "@itwin/workspace-editor",
+      "projectFolder": "utils/workspace-editor",
```

Comment:
can we put this in the `tools` folder instead of creating a new one?

Reply:
No, I don't think that's right. `tools` is only for developers. 

This is intended to be a desktop utility for users (admins, actually). I don't know if we have any other utilities yet, but if we do they should also be npm packages, I guess.